cypress/e2e/webTables.cy.js

```diff
@@ -1,5 +1,91 @@
+/// <reference types='cypress' />
+
 describe('Web Tables page', () => {
-  it('', () => {
-    
+  const testData = {
+    firstName: 'test_firstName',
+    lastName: 'test_lastName',
+    email: 'test_555@qa.team',
+    age: '26',
+    salary: '36000',
+    department: 'Testing'
+  };
+
+  
+
+  beforeEach(() => {
+    cy.visit('https://demoqa.com/webtables/');
+  });
+
+  it('The pagination section is present on the Web Tables page', () => {
+    cy.get('.-pagination').should('exist');
+  });
+
+  it('Select the number of rows per page', () => {
+    cy.get('[aria-label="rows per page"]').select('5 rows');
+  });
+
+  it('It is possible to add a new record to the table', () => {
+    cy.get('#addNewRecordButton').click();
+
+    cy.get('.modal-content').should('exist');
+
+    cy.get('#firstName').type(testData.firstName);
+    cy.get('#lastName').type(testData.lastName);
+    cy.get('#userEmail').type(testData.email);
+    cy.get('#age').type(testData.age);
+    cy.get('#salary').type(testData.salary);
+    cy.get('#department').type(testData.department);
+
+    cy.get('#submit').click();
+
+    cy.get('.rt-tbody').should('contain', testData.firstName);
+  });
+
+  it('It is possible to delete a record from the table', () => {
```

Comment:
you have to implement a function here that will delete all of the recordings no matter how many of them it is there. think about the loop